docs/quickstart/cross-chain-transfers.md

```diff
@@ -0,0 +1,298 @@
+# Transfer AVAX Tokens Between Chains
+
+## Introduction
+
+This article shows how to transfer AVAX tokens programmatically between any two chains of the Primary Network.
+
+If you are looking for how to transfter AVAX tokens using the web wallet, please check out [this article](https://support.avax.network/en/articles/6169872-how-to-make-a-cross-chain-transfer-in-the-avalanche-wallet).
+
+## Requirements
+
+You've completed [Run an Avalanche Node](../nodes/build/run-avalanche-node-manually.md) and are familiar with [Avalanche's architecture](../overview/getting-started/avalanche-platform.md). You are familiar with the [AvalancheJS](https://github.com/ava-labs/AvalancheJS) repo.
+
+To use AvalancheJS, you can clone the repo:
+```zsh
+$ git clone https://github.com/ava-labs/avalanchejs.git
+```
+
+or add it to an existing project:
+
+```zsh
+$ yarn add --dev avalanche
+```
+For this tutorial we will use a cli to run the example scripts directly from an AvalancheJS directory.
+
+
+In order to send AVAX, you need to have some AVAX! You can get real AVAX by buying it on an exchange, or you can get testnet AVAX from the [AVAX Test Faucet](https://faucet.avax.network), which is a free and easy way to get to play around with Avalanche.
+
+
+## Transferring AVAX using AvalancheJS
+
+The easiest way to transfer AVAX between chains is to use [AvalancheJS](https://github.com/ava-labs/AvalancheJS) which is a programmatic way to access and move AVAX.
+
+AvalancheJS allows you to create and sign transactions locally which is why it is the recommended way to transfer AVAX between chains. We are moving away from using AvalancheGo's keystore because it requires you to keep your keys on a full node which makes them a target for malicious hackers.
+
+You can manage private keys you use with AvalancheJS directly in the AvalancheJS directory by doing the following:
+
+Rename [``secrets.example``](https://github.com/ava-labs/avalanchejs/blob/master/examples/secrets.example) to ``secrets.json``
+
+Add your Private Key as an object
+```json
+{
+  "newPassword": "newPassword",
+  "oldPassword": "oldPassword",
+  "password": "password",
+  "token": "token",
+  "user": "user",
+  "privateKey": "PrivateKey-ewoqjP7PxY4yr3iLTpLisriqt94hdyDFNgchSxGGztUrTXtNN"
```

Comment:
are newPassword/oldPassword/password/token/user needed for this tutorial? If not, please remove them.

Reply:
I believe those were needed for the Keystore API, which we don't use any more. So definitely no need to have them.